source/canvas.ts

```diff
@@ -245,6 +249,20 @@ export class Canvas extends Resizable {
         this.onResize(); // invokes frameScaleNext and frameSizeNext
     }
 
+    /**
+     *
+     */
+    protected configureContextLostAndRestore(): void {
+        this._lostContextExtension = this._context.gl.getExtension('WEBGL_lose_context');
```

Comment:
I cannot find if that extension is supported on all browsers, specifically IE and Safai. What happens when this extension is not supported?

Reply:
nvm, I just saw `testLoseContext()`